acceptance-tests/src/test/resources/features/site/publication.feature

```diff
@@ -48,3 +48,15 @@ Scenario: Display multiparagraph key facts
         # bullet list
         | Morbi eget congue turpis. Fusce in purus mollis ...           |
         | Aliquam erat volutpat. Quisque eget massa ...                 |
+
+Scenario: Show new v3 attachments with text along with old v2 attachments
```

Comment:
Reference to this scenario needs to be added to the JIRA ticket referencing all bits of code that need removing once the data upgrade scripts are run (see comment in the review of `Publication.java#getAttachmentsOld`).

Reply:
Created RPS-238